README.md

```
@@ -1,6 +1,6 @@
 # The KYVE Network
 
-###### v1.5.0
+###### v2.0.0
```

Markdownlint (0.37.0)

Expected: h2; Actual: h6
Heading levels should only increment by one level at a time

(MD001, heading-increment)